sympy/series/series.py

```diff
@@ -61,3 +62,110 @@ def series(expr, x=None, x0=0, n=6, dir="+"):
     """
     expr = sympify(expr)
     return expr.series(x, x0, n, dir)
+
+
+def inversion_series(f, x, a=0, n=3):
+    r"""
+    The Lagrange inversion theorem (or Lagrange inversion formula, which we abbreviate
+    as LIT), also known as the Lagrange--Bürmann formula, gives the Taylor series
+    expansion of the inverse function of an analytic function.
+    It finds important applications in combinatorics and calculus.
```

Comment:
The docstring should open with a one line single sentence description of what the function does or what it returns.

Reply:
@oscarbenjamin Dear Oscar, these lines have been in the old PR for a while. You reviewed it many times. Please, for future, try go give a comprehensive feedback. Thanks. From my side, I will rewrite accordingly.

Reply:
> the old PR which you reviewed many times

I did not review it: I asked others to review it and made a few comments that the author did not seem to understand. 
In any case I am not going to write an exhaustive list of what needs to be done any time I review anything because that would be more work than just writing all of the code myself.

Reply:
I am not asking for lots of feedback, but for targeted feedback. I don't want you to comment on doc strings in first place (keep in mind that we both volunteer), I rather need your knowledge on bridging to ring series (and appreciate the concrete examples!).